CHANGELOG.md

```diff
@@ -50,6 +50,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Migrate from vue-cli to vite
```

Comment:
As we discussed in a previous daily, it would be nice to include the Github issue number for this change ?

Also, regarding chronological order, could we have the latest change at the bottom (or on top) of the list? I'm okay with either way but just want to have some common rule since some of them are placed in different order now.

Reply:
There is no github issue for this, and we had an agreement last month that we would not create new issues on github.

Regarding the order, I agree that it's good to agree on how to do it. I saw someone else starting to write at the top, and did the same.

Reply:
For the Github issue, I mean it could also be the PR number, in this case it is 982, if that makes sense :)